src/data/mod_json.rs

```diff
@@ -164,17 +164,18 @@ impl ModJson {
         let file = std::fs::File::create(path).expect("create failed");
         serde_json::to_writer_pretty(file, self).expect("Write failed");
     }
-
 }
 
 impl From<SharedPackageConfig> for ModJson {
     fn from(mut shared_package: SharedPackageConfig) -> Self {
+        // Only bundle mods that are not specifically excluded in qpm.json
         shared_package
             .restored_dependencies
-            // keep if header only is false, or if not defined
-            .retain(|dep| !dep.dependency.additional_data.headers_only.unwrap_or(false));
+            // keep if include_qmod is not defined or true
```

Comment:
this should remain the old behaviour of not including if not defined& header only

Reply:
it at least should definitely not be the default to include the qmod if inclusion not defined & the package is header only